package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bs-moment",
-  "version": "0.4.2",
+  "version": "0.4.3",
```

Comment:
Should I increase version ? Or it will be increase before publication to npm by person doing publication?
How it works?

Reply:
@kamilkuz don't worry - i'll bump version after merge

Reply:
@Jimexist 
Ok, I set the previous version number